odoo-11/odoo-prod/odoo.conf

```
@@ -0,0 +1,64 @@
+[options]
+addons_path = /opt/odoo/odoo/addons,/opt/odoo/custom-addons/account-financial-reporting,/opt/odoo/custom-addons/account-financial-tools,/opt/odoo/custom-addons/account-invoicing,/opt/odoo/custom-addons/backend_theme,/opt/odoo/custom-addons/bank-payment,/opt/odoo/custom-addons/commission,/opt/odoo/custom-addons/community-data-files,/opt/odoo/custom-addons/l10n-spain,/opt/odoo/custom-addons/mis-builder,/opt/odoo/custom-addons/Odoo-Material-Theme,/opt/odoo/custom-addons/purchase-workflow,/opt/odoo/custom-addons/reporting-engine,/opt/odoo/custom-addons/sale-workflow,/opt/odoo/custom-addons/server-ux,/opt/odoo/custom-addons/slv-hr,/opt/odoo/custom-addons/web,/opt/odoo/custom-addons/jackjack82,/opt/odoo/custom-addons/slv,/opt/odoo/custom-addons/server-tools,/opt/odoo/custom-addons/server-env
+data_dir = /opt/odoo/odoo
```

I've found this is a bad idea, because data Odoo folder (actually `filestore` and `sessions` folders) is located within Odoo repository, and have no sense.

This is an inheritance from Crt-based Dockerfile, not ours.

Better this (that probably implies folder creation in `Dockerfile`):
```suggestion
data_dir = /opt/odoo/data
```